go/apps/http_api_nostream/tests/test_resource.py

```diff
@@ -81,39 +84,88 @@ def test_many_errors(self):
         self.assert_no_attribute(opts, "odd")
 
 
+class TestMsgCheckHelpers(TestCase):
+    def test_is_unicode(self):
+        self.assertTrue(MsgCheckHelpers.is_unicode(u"123"))
+        self.assertFalse(MsgCheckHelpers.is_unicode(None))
+        self.assertFalse(MsgCheckHelpers.is_unicode("123"))
```

Comment:
`self.assertEqual(..., True)` is probably better here, because we are explicitly looking at the return value. (Also, we get much nicer failure messages.) `assertTrue()` and friends are harder to get decent output from, but are useful when an equality test isn't possible.